azure-mgmt-resources/src/test/java/com/azure/management/resources/core/TestUtilities.java

```diff
@@ -35,4 +37,8 @@ public static void sleep(int milliseconds, boolean isRecordMode) {
         }
         return res;
     }
+
+    public static <T> boolean isEmpty(PagedIterable<T> iterable) {
```

Comment:
Would better to use just `Iterable`? More general type.

Reply:
Well, I thinking of provide a PagedList for this in future to provide back compatibility.